pages/1.12/installing/production/advanced-configuration/configuration-reference/index.md

```diff
@@ -95,6 +95,7 @@ The configuration parameters for [DC/OS Open Source](https://dcos.io/) are [here
 | ca_certificate_path                   | Use this to set up a custom CA certificate. See [using a Custom CA Certificate](/1.11/security/ent/tls-ssl/ca-custom#configuration-parameter-reference) page for a detailed configuration parameter reference. [enterprise type="inline" size="small" /] |
 | ca_certificate_key_path           | Use this to set up a custom CA certificate. See [using a Custom CA Certificate](/1.11/security/ent/tls-ssl/ca-custom#configuration-parameter-reference) page for a detailed configuration parameter reference. [enterprise type="inline" size="small" /] |
 | ca_certificate_chain_path       | Use this to set up a custom CA certificate. See [using a Custom CA Certificate](/1.11/security/ent/tls-ssl/ca-custom#configuration-parameter-reference) page for a detailed configuration parameter reference. [enterprise type="inline" size="small" /] |
+| [permissions_cache_ttl_seconds](#permissions-cache-ttl-seconds)   | [enterprise type="inline" size="small" /] The maximum number of seconds after a permission is revoked before it is guaranteed to not apply. |
```

Comment:
I think is easier to read if you just use the same first sentence as in the change [below](https://github.com/mesosphere/dcos-docs-site/pull/1342/files#diff-3c282e2dc53322cebc0d7f0a2835233aR493), since: It doesn't only works for revocations (also granting will take the ttl seconds before is propagated). And it just reads easier.